nagios/check.py

```diff
@@ -172,6 +176,27 @@ def check(self, instance):
             raise Exception('No Nagios configuration file specified')
         for tailer in self.nagios_tails[instance_key]:
             tailer.check()
+        i_key = {"type": self.INSTANCE_TYPE, "url": instance.get("nagios_conf")}
```

Comment:
Is the URL unique enough? Looking at https://github.com/StackVista/sts-agent-integrations-core/blob/master/nagios/conf.yaml.example the nagios_conf setting is a path to Nagios' configuration file, it may be the same across installations. Is this the kind of uniqueness okay for the product owner?

Reply:
added the hostname of the nagios agent in `instance-key` as well.